docs/src/main/tut/typeclasses/semigroup.md

```diff
@@ -7,92 +7,164 @@ scaladoc: "#cats.kernel.Semigroup"
 ---
 # Semigroup
 
-A semigroup for some given type A has a single operation
-(which we will call `combine`), which takes two values of type A, and
-returns a value of type A. This operation must be guaranteed to be
-associative. That is to say that:
+If a type `A` can form a `Semigroup` it has an **associative** binary operation.
 
-```scala
-((a combine b) combine c)
+```tut:book:silent
+trait Semigroup[A] {
+  def combine(x: A, y: A): A
+}
 ```
 
-must be the same as
+`combine` also needs to be associative, which means the following equality must hold for any choices of `x`, `y`, and
+`z`.
 
-```scala
-(a combine (b combine c))
+```
+combine(x, combine(y, z)) = combine(combine(x, y), z)
+```
+
+A common example of a semigroup is the type `Int` with the operation `+`.
+
+```tut:reset:book:silent
+import cats.Semigroup
+
+implicit val intAdditionSemigroup: Semigroup[Int] = new Semigroup[Int] {
+  def combine(x: Int, y: Int): Int = x + y
+}
+
+val x = 1
+val y = 2
+val z = 3
+```
+
+```tut:book
+Semigroup[Int].combine(x, y)
+
+Semigroup[Int].combine(x, Semigroup[Int].combine(y, z))
+
+Semigroup[Int].combine(Semigroup[Int].combine(x, y), z)
+```
+
+Infix syntax is also available for types that have a `Semigroup` instance.
+
+```tut:book
+import cats.syntax.semigroup._
+
+1 |+| 2
```

Comment:
The issue I had with introducing the `Option[A]` `Semigroup` this early was there are some nuances to it with regards to how it handles `None`. Since the `Semigroup` instance is sort of incidental because it really is a `Monoid` instance, I opted to introduce it in the `Monoid` section instead.

 I could perhaps show an early `Map` example and say details to come later in this post. As I type that out I think that's a good idea, I'll go ahead and do that.